tests/CsvReadTest.php

```diff
@@ -398,13 +283,174 @@ public function testSkipsOverflow()
     {
         $fileName = __DIR__ . '/data/simple.csv';
 
-        $csvFile = new CsvFile(
+        $csvFile = new CsvReader(
             $fileName,
-            CsvFile::DEFAULT_DELIMITER,
-            CsvFile::DEFAULT_ENCLOSURE,
-            CsvFile::DEFAULT_ENCLOSURE,
+            CsvReader::DEFAULT_DELIMITER,
+            CsvReader::DEFAULT_ENCLOSURE,
+            CsvReader::DEFAULT_ENCLOSURE,
             100
         );
         self::assertEquals([], iterator_to_array($csvFile));
     }
+
+    public function testException()
+    {
+        try {
+            $csv = new CsvReader(__DIR__ . '/nonexistent.csv');
+            $csv->getHeader();
```

Comment:
Nevyhodí to exception i bez toho načtení headeru teď?

Reply:
copy & paste :( bohuzel to tam bylo uz z predchoziho pr :(

fixed